recipes/opentelemetry-api/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "opentelemetry-api" %}
+{% set version = "0.12b0" %}
```

Comment:
Are there full releases @mariusvniekerk? Otherwise this should follow CFEP5

Reply:
there aren't yet.  They seem to be using the b tag to prevent automatic installation until they hit a 1.0

Reply:
https://pypi.org/project/opentelemetry-api/#history

Reply:
So I think this is blocked then

Reply:
It's basically the same as `black` which has also never had a non-beta release